### ISSUE TITLE
several issues involving action toggle-preview-wrap

### ISSUE THREAD

**Issue by step-** [junegunn/fzf#2336]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [x] bash
    - [x] zsh
    - [ ] fish
    
## Problem / Steps to reproduce

This is how I'm testing preview wrapping
```sh
fzf --preview="cat {}" --bind "ctrl-w:toggle-preview-wrap"
```

In versions 0.25.0 and master pressing ctrl-w produces no effect; the action doesn't seem to work at all.

In version 0.23.1 it's working as expected, meaning:
- ctrl-w is a toggle (wrap/unwrap)
- wrapping spaces all lines correctly, including blank lines
- unwrapping spaces all lines correctly, including blank lines

In versions 0.24.{0..4} I have seen two different issues depending on the version
- ctrl-w wraps once then it doesn't unwrap (no toggle)
- wrapping spaces line incorrectly, meaning that an existing blank line is written over by the wrapped line above it

In summary, action toggle-preview-wrap has stopped working and the last good version was 0.23.1.

**Comment by junegunn** [comment]
Thanks for the report. Should be fixed on master!

**Comment by step-** [comment]
Fix confirmed on release 0.25.1.   Thank you!
